### ISSUE TITLE
Splatmoji [Feature Req]

### ISSUE THREAD

**Issue by sakkamade** [owl4ce/dotfiles#125]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when there are so many kaomoji out there, but it takes so much time to type them in.

**Describe the solution you'd like**
I want to suggest to add [splatmoji](https://github.com/cspeterson/splatmoji) to your default environment because it is just amazing.

**Describe alternatives you've considered**
I've considered, and actually used, [kao](https://github.com/fand/kao) and [oji](https://github.com/xxczaki/oji) for quite a some time, but after I moved to Linux, and after I made some time to look into this one, it had out pushed them plainly! :heart: 

**Additional context**
In short, it can use rofi, and this is what makes it amazing.

**Comment by owl4ce** [comment]
Looks good, I'll try it later if it fits to my environment.
